app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -1,13 +1,10 @@
 <?php
-
```

Comment:
Please do not remove the space here; the space is intended so that it will be easier to read the code.